src/components/00-materials/icons-raw/axa-logo-open.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg viewBox="0 0 80 80" xmlns="http://www.w3.org/2000/svg" width="24" height="24" ><g fill="none"><path fill="#FF1721" d="M49.026 39.457L77.948 1.9h-4.272L44.658 39.457z"/><path d="M0 0v79.85h79.85V0H0zm77.948 77.948H1.901v-7.604h3.353c.042-.293 1.92-2.99 2.177-3.247.254-.262 6.7-8.731 6.7-8.731h11.303l1.323 4.58s-3.3 4.312-4.186 5.353c-.525.615-2.027 2.045-2.027 2.045h5.317s.183-.44.609-1.231c.428-.79 1.933-2.618 1.933-2.618s.658 2.047.71 2.392c.067.429.247 1.457.247 1.457h5.376s.19-.409.618-1.199c.424-.79 1.982-2.498 1.982-2.498s.597 1.895.65 2.24c.068.429.246 1.457.246 1.457h6.872s-1.027-1.781-1.26-2.376c-.133-.33-2.284-7.063-2.284-7.063l1.791-2.54h11.39s2.594 8.345 2.903 9.448c.202.83.315 1.678.336 2.531h7.171c-1.086-.769-3.817-10.42-5.143-14.128l-3.474-12.04c-.169-.458-.297-2.115-.297-2.115h-5.549c-.297.791-.804 1.518-1.149 2.157-.427.792-7.476 10.035-9.025 11.998-.228.292-.36.455-.36.447 0-.018-.049-.191-.118-.447-.256-.89-.78-2.676-.78-2.676s6.006-7.633 6.62-8.41c1.824-2.308 2.865-3.069 2.865-3.069h-5.687a3.783 3.783 0 0 1-.575 1.736c-.273.419-4.928 6.255-4.928 6.255s-1.464-5.28-1.741-6.135a5.947 5.947 0 0 1-.223-1.856h-6.826c.257.292 1.045 2.241 1.045 2.241l3.365 10.36-1.968 2.517c-.084-.126-3.645-11.808-3.797-12.295-.17-.551-.35-2.823-.35-2.823h-5.844s-1.135 2.47-1.922 3.613c-.325.473-4.03 5.437-7.978 10.542-4.393 5.854-9.176 11.791-9.28 11.867a1.6 1.6 0 0 0-.13.128V1.9h76.046v76.047zM22.64 49.503c.256 1.022 1.727 5.555 1.727 5.555h-7.595s4.796-5.838 5.61-7.59c.09-.195.108-.044.108.049a8.11 8.11 0 0 0 .15 1.986zm9.876 11.64l2.022-2.622s1.177 3.57 1.37 3.777l-2.09 2.872-1.302-4.027zm19.304-11.64c.257 1.022 1.729 5.555 1.729 5.555h-7.595s4.796-5.838 5.61-7.59c.09-.195.106-.044.106.049a8.111 8.111 0 0 0 .15 1.986z" fill="#00008F"/></g></svg>
```

Comment:
Im not sure but i think also the viewbox should be adapted so that matches 1:1 to its size. otherwise we have a "zoomed" out SVG. Please double check my hypothesis.

So to be clear:
IS: `<svg viewBox="0 0 80 80" xmlns="http://www.w3.org/2000/svg" width="24" height="24" >`
SHOULD: `<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="24" height="24" >`

Reply:
Thats with viewBox `0 0 24 24`.
![Bildschirmfoto 2019-09-10 um 13 51 40](https://user-images.githubusercontent.com/38284563/64611608-26428f80-d3d2-11e9-97b6-c0b632300dfd.png)
In my opinion this only works when the icon is actually 24x24 big. So either let the viewbox be as it is, or make a new icon with the size of 24x24.

Reply:
ok but seems then is not exported correctly from the designers